File: app/src/main/java/com/essamhshmey/question/repository/CalculateAgeRepository.java
package com.essamhshmey.question.repository;

import androidx.lifecycle.MutableLiveData;

import com.essamhshmey.question.model.Person;

public abstract class CalculateAgeRepository {

  abstract MutableLiveData<Person> calculateYourAge(String name, Integer age);

}

File: app/src/main/java/com/essamheshmy/calculate_your_age_with_mvvm/repository/CalculateAgeRepository.kt
package com.essamheshmy.calculate_your_age_with_mvvm.repository

import androidx.lifecycle.MutableLiveData
import com.essamheshmy.calculate_your_age_with_mvvm.model.entity.Person


interface CalculateAgeRepository {
    fun calculateYourAge(name: String, age: Int):Person
}